aws-rds-globalcluster/aws-rds-globalcluster.json

```diff
@@ -57,7 +57,8 @@
   "additionalProperties": false,
   "createOnlyProperties": [
     "/properties/GlobalClusterIdentifier",
```

Comment:
what about `Engine`, `EngineVersion`?

Reply:
I think we need to discuss it, I will ping you and Pablo on slack.